seed/challenges/01-front-end-development-certification/intermediate-bonfires.json

```diff
@@ -352,6 +352,7 @@
             "Traduce la cadena de texto que se te provee al Latín de los cerdos (Pig Latin)",
             "<a href=\"http://en.wikipedia.org/wiki/Pig_Latin\" target=\"_blank\">Pig Latin</a> toma la primera consonante (o grupo de consonantes) de una palabra en inglés, la mueve al final de la palabra y agrega un \"ay\".",
```

Comment:
I don't know if this is proper place for this comment, but maybe the wiki article should point to a Spanish version? [Pig Latin en Espanol](https://es.wikipedia.org/wiki/Pig_Latin)